uint16[] id_list
float32[] input_pos_list
float32[] tt_vel_limit_list
float32[] tt_accel_limit_list
float32[] tt_decel_limit_list
